environments/echis/app-processes.yml

```diff
@@ -1,77 +1,117 @@
-jython_memory: "1024m"
-formplayer_memory: "1024m"
+jython_memory: "12288m"
+formplayer_memory: "12288m"
 # workers = static_factor + (factor * cpu_count)
 gunicorn_workers_factor: 0
 gunicorn_workers_static_factor: 6
 
 celery_processes:
-  None:
-    repeat_record_queue:
-    async_restore_queue:
   echis_server0:
-    celery,export_download_queue,email_queue,case_import_queue:
-      concurrency: 1
-    background_queue,case_rule_queue,analytics_queue:
-      concurrency: 1
     beat: {}
-    celery_periodic:
+    flower: {}
+    export_download_queue:
+      concurrency: 5
+    email_queue:
+      pooling: gevent
+      concurrency: 50
+      num_workers: 2
+    case_import_queue:
+      concurrency: 4
+    celery:
+      concurrency: 4
+    saved_exports_queue:
+      concurrency: 3
+      max_tasks_per_child: 3
       concurrency: 1
+      optimize: True
+  echis_server3:
     saved_exports_queue:
+      concurrency: 3
+      max_tasks_per_child: 3
       concurrency: 1
       optimize: True
+    celery_periodic:
+      concurrency: 4
     submission_reprocessing_queue:
-      concurrency: 1
+      concurrency: 3
     reminder_case_update_queue:
       pooling: gevent
-      concurrency: 5
+      concurrency: 3
     reminder_queue:
       pooling: gevent
       concurrency: 5
-    reminder_rule_queue,ucr_indicator_queue,ucr_queue:
-      concurrency: 1
+    reminder_rule_queue:
+      concurrency: 3
+      max_tasks_per_child: 3
     sms_queue:
       pooling: gevent
+      concurrency: 5
+    sumologic_logs_queue:
```

Comment:
[@demis-svenska You can move this under `None` (like this](https://github.com/dimagi/commcare-cloud/blob/master/environments/icds-cas/app-processes.yml#L6)) because it's not needed for eCHIS env.

Reply:
ok